README.beginner-developer.md

```diff
@@ -0,0 +1,76 @@
+# Tlf ham radio contest logger
```

Comment:
At this line in README.md, there is a link to http://travis-ci.org , no idea what for, Anyway, now I get

`Please be aware travis-ci.org will be shutting down in several weeks, with all accounts migrating to travis-ci.com.`

Could somebody please resolve this case please...

Reply:
see https://github.com/Tlf/tlf/pull/246#discussion_r582277343, commented the wrong thread